sale_order_import_edifact/wizard/sale_order_import.py

```diff
@@ -0,0 +1,236 @@
+import json
+import logging
+import os
+from base64 import b64decode
+from collections import defaultdict
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+logger = logging.getLogger(__name__)
+
+
+class SaleOrderImportEdifact(models.TransientModel):
+    _name = "sale.order.import"
+    _inherit = ["sale.order.import", "base.edifact"]
```

Comment:
pls avoid extending base.edifact. That's a tool, not a mixin and should be used directly where needed.
We already removed base.business.document.import inheritance a long time ago ;)